src/main/java/at/hannibal2/skyhanni/config/features/inventory/GetFromSackConfig.java

```diff
@@ -23,4 +24,14 @@ public class GetFromSackConfig {
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean superCraftGFS = true;
+
+    @Expose
+    @ConfigOption(name = "Default GfS", desc = "If you don't provide an amount, a default one will be used instead.")
```

Comment:
Please correct my bad writing, but add something like that.
```suggestion
    @ConfigOption(name = "Default GfS", desc = "If you don't provide an amount, a default one will be used instead. Queued Gfs needs to be turned on for this to work.")
```